tests/ui/test_discopane.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from pages.discopane import DiscoveryPane
+
+@pytest.fixture(scope='session')
+def base_url(base_url):
+    # This is hardcoded for now, once we go through about:addons, we'll be able to use the template expansion
+    if base_url[len(base_url)-1] != '/':
+        base_url += '/'
```

Comment:
Let's assume there is no trailing slash on the base URL. This is how we do it for other suites.